app/views/layouts/application.html.haml

```diff
@@ -7,6 +7,7 @@
     = csrf_meta_tag
     = stylesheet_link_tag "application"
     = javascript_include_tag "application"
+    = content_for :head
```

Comment:
I think you have to `yield :head` here instead, you use `content_for :head` in the templates and yield in layout here.